src/components/map/ha-locations-editor.ts

```diff
@@ -199,15 +204,21 @@ export class HaLocationsEditor extends LitElement {
     }
   }
 
-  private _updateLocation(ev: DragEndEvent) {
-    const marker = ev.target;
-    const latlng: LatLng = marker.getLatLng();
-    let longitude: number = latlng.lng;
+  private normalizeLongitude(longitude: number): number {
```

Comment:
```suggestion
  private _normalizeLongitude(longitude: number): number {
```

Reply:
btw. I don't like this style personally but if we are going to enforce it, we should just a lint rule for it